.github/workflows/lint-test.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
         run: ct lint --config ct.yaml
 
       - name: Create kind cluster
-        uses: helm/kind-action@v1.2.0
+        uses: helm/kind-action@v1.4.0
```

Comment:
Need to update this action to get tests to pass. Latest keda charts require an updated version of kubernetes.